cmd/common/utils/parser-tableformatter.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"fmt"
+
+	"github.com/kinvolk/inspektor-gadget/pkg/columns"
+	"github.com/kinvolk/inspektor-gadget/pkg/columns/formatter/textcolumns"
+)
+
+const (
+	KubernetesTag       string = "kubernetes"
+	ContainerRuntimeTag string = "runtime"
+)
+
+var SupportedTags = map[string]struct{}{
+	KubernetesTag:       {},
+	ContainerRuntimeTag: {},
+}
+
+type Option func(*GadgetOptions) error
+
+func WithMetadata(metadataTag string) Option {
+	return func(opts *GadgetOptions) error {
+		if _, ok := SupportedTags[metadataTag]; !ok {
+			return fmt.Errorf("invalid metadata tag: %q", metadataTag)
+		}
```

Comment:
Do we really need to check this?

Reply:
When I did it, I was expecting that we would be able to verify the tags used in the event structs, but I just realized it is impossible. I removed it.